### PR TITLE
KRP-1657 add refund direct debit endpoint

### PR DESCRIPTION
**Pull request by haipham-kontist**
Add refund direct debit endpoint. It doesn't need to return or save something.

### CODE REVIEW

File: src/routes/transactions.ts [hunk 1]
```diff
@@ -343,3 +343,13 @@ export const creteBookingFromReservation = (person, reservation, incoming?) => {
...
 };
+
+export const directDebitRefund = (req, res) => {
+  log.info("refund confirmed", {
+    personId: req.params.person_id,
+    accountId: req.params.account_id,
+    bookingId: req.body.booking_id,
+  });
+
+  res.status(200).send("OK");
```
Comment (p-janik) [inline src/routes/transactions.ts]: I would either send status 204 or empty JSON object, best to check with docs. Returning plain string to a caller who expects JSON might result in error depending on response parsing implementation

Comment (haipham-kontist) [reply src/routes/transactions.ts]: thanks, I return 201 as in the doc.

**Review by p-janik — APPROVED** [review]
Looks good 👍

**Review by ogulcancelik — APPROVED** [review]
👍

**Pull request merged**